packages/shared/src/components/notifications/FirstNotification.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { ReactElement, useEffect } from 'react';
+import { NotificationType } from '../../graphql/notifications';
+import usePersistentContext from '../../hooks/usePersistentContext';
+import NotificationItem from './NotificationItem';
+import { NotificationIcon } from './utils';
+
+const READ_KEY = 'FIRST_NOTIFICATION_READ';
+
+function FirstNotification(): ReactElement {
+  const [isUnread, setIsUnread] = usePersistentContext(READ_KEY, true);
+
+  useEffect(() => {
+    return () => {
+      setIsUnread(false);
+    };
+  }, []);
+
+  return (
+    <NotificationItem
+      isUnread={isUnread}
+      type={NotificationType.System}
+      icon={NotificationIcon.Bell}
+      targetUrl="#"
```

Comment:
Before merging, make the URL be pointed at our changelog for the notifications system.

Reply:
Reminder

Reply:
Another reminder before merging 🥳